### DISCUSSION TITLE
Is Path segment broken on 7.55.0?

### DISCUSSION THREAD

**Discussion by jvsca** [JanDeDobbeleer/oh-my-posh#2009] [General]
Before 7.55.0:
`C:\`

After  7.55.0:
`C:" --pswd=C:" `

**Comment by JanDeDobbeleer** [comment]
@jvsca so to be clear, this is on the root, right?

**Reply by jvsca** [reply]
Yes, root of any drive.
Same in version 7.55.1.

**Reply by JanDeDobbeleer** [reply]
@jvsca fixed in #2011

**Reply by jvsca** [reply]
@JanDeDobbeleer Thank you.
